docs/closepseudoconsole.md

```diff
@@ -48,6 +48,8 @@ Remarks
 
 Upon closing a pseudoconsole, client applications attached to the session will be terminated as well.
 
+A final painted frame may arrive on `hOutput` from the pseudoconsole when this API is called. It is expected that the caller will drain this information from the communication channel buffer and either present it or discard it. Failure to drain the buffer may cause the Close call to wait indefinitely until it is drained or the communication channels are broken another way.
```

Comment:
Wait are we keeping this behavior? I thought we were removing this with your PR. Or should we add a "On Windows builds prior to N," to this?

Reply:
I wasn't going to add "on Windows builds prior to N" because most people's knee-jerk reaction to that is to start version-testing the OS.

As long as you write your software defensively and are ready to accept a final painted frame on `hOutput`, you will be fine for the entire timeline before and after my fix. That's what I'd prefer to advocate in the documentation.

Reply:
Sounds good to me